marlowe-playground-client/src/Simulation/BottomPanel.purs

```diff
@@ -377,6 +384,22 @@ panelContents state MarloweErrorsView =
     in
       { message, startColumn, startLineNumber, firstLine, restLines }
 
+panelContents state MarloweLogView =
+  section
+    [ classes [ ClassName "panel-sub-header", aHorizontal, Classes.panelContents, flexLeft ]
+    ]
+    content
+  where
+  inputLines = state ^.. (_marloweState <<< traversed <<< _currentTransactionInput <<< to txInputToLines)
+
+  content =
+    [ div [ classes [ ClassName "error-headers", ClassName "error-row" ] ]
+        [ div [] [ text "Action" ]
+        , div [] [ text "Slot" ]
+        ]
+    , ul [] (reverse (fold inputLines))
+    ]
```

Comment:
I think an HTML `<table>` would be appropriate here if we have a header and everything (probably for state too, but that was a different PR)

Reply:
This is a CSS question, it's what was implemented by someone better than me with these things for the "Current State". Besides, it's barely a data table since it has only 2 columns and a list makes just as much sense to me. It is a reasonable question to ask though, part of the problem with the table for current state is that there are a variable number of columns, a table would be easier here.

Reply:
I think it could be considered a list if it didn't have headers, but the fact that headers and the cells need to be aligned smells like table... Anyway, it is not important for this PR, but I think the reason it is not a table probably has to do with this discussion (https://stackoverflow.com/questions/17503456/is-it-good-to-use-tables-in-html-5), so if you find yourself fighting with CSS to get some of these aligned I would just make it a table.

Reply:
While I agree that a table would make sense, I would say that a list also makes sense. Again, I'd be happy if someone changes it in a future PR but I don't feel strongly enough about it to feel it needs to be changed in this one.